### PR TITLE
Add support for height prop in pie charts

### PR DESCRIPTION
**Pull request by maricris-sn**
#### Screens

pie-chart-heights

#### Breaking Changes

[No] breaking changes.  Just adding a height prop support for pie charts which can be optional.  Default still set to 400px.

#### Runway Ticket URL

We have changes we need to put in place for Project Launch Dashboard which requires the pie charts to appear in a certain height.  Adjusting the heights however cannot be done since the graphs are rendered on the fly and CSS can't affect the rendered SVG viewBox.

Related Runway Card: [TTNS-723](https://nitro.powerhrg.com/runway/backlog_items/TTNS-723)
Related Huddle: [Project Launch Dashboard](https://huddle.powerapp.cloud/projects/315/handoffs/577)

#### How to test this

_Part 1_
- create a Circle Chart (either in React or in Rails)
- don't specify a height prop.  rendered chart should have a height of 400px in viewBox

_Part 2_
 - create a Circle Chart (either in React or in Rails)
 - pass a specific prop in string (ie., 200 -- for 200px). rendered chart should have a height of 200px in viewBox

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_circle_chart/_circle_chart.jsx [hunk 1]
```diff
@@ -43,6 +44,7 @@ const CircleChart = (props: CircleChartProps) => {
     dataLabelHtml = '<div>{point.name}</div>',
     dataLabels = false,
     headerFormat = null,
+    height = '400',
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_circle_chart/_circle_chart.jsx:47]: @maricris-sn Can this unset by default? see https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_bar_graph/_bar_graph.jsx

Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/pb_circle_chart/_circle_chart.jsx:47]: +1 ^^ Per highcharts docs, it's best to leave it unset. So you can just make this line say `height,`.

Comment (maricris-sn) [reply playbook/app/pb_kits/playbook/pb_circle_chart/_circle_chart.jsx:47]: @jasoncypret @jasperfurniss ah, right. yes, I can have this unset.  It's defaulting to 400 anyway as per highcharts.

**Comment by jasoncypret** [comment]
@maricris-sn going to fix up your PR

**Review by jasperfurniss — APPROVED** [review]

File: playbook/app/pb_kits/playbook/pb_circle_chart/circle_chart.rb [hunk 2]
```diff
@@ -25,6 +25,7 @@ class CircleChart < Playbook::KitBase
       prop :use_html, type: Playbook::Props::Boolean, default: false
       prop :legend, type: Playbook::Props::Boolean, default: false
       prop :title, default: ""
+      prop :height, default: "400"
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_circle_chart/circle_chart.rb:28]: @maricris-sn Can we leave this unset as well?

**Pull request merged**